jumpstart-week2/README.md

```diff
@@ -0,0 +1,222 @@
+
+##
+
+To tackle sizing and positioning of elements, you first have to understand the "box model".
+
+Every HTML element has several boxes around it whose dimensions can be controlled through CSS. Every element has all of these boxes around it, though often these boxes can coincide.
+
+The innermost box is the "content box". This is the space that all the children of that element occupy. 
+
+The next box is the "border box". This includes all the content of the content box PLUS the border of the element.
+
+Then is the "padding box". This includes the border box PLUS some padding (empty space) that the element should be surrounded by.
+
+Finally is the "margin box". This includes the padding box PLUS the margins around the element. Margins and padding both define empty space that should surround elements. The main difference is that two adjacent elements can share the same margins, but they can't share the same padding.
+
+For each of these boxes, you can define a width and a height with a corresponding CSS property, these values change the difference between the box and the next inner box (i.e. `border-width: 10px` makes the border box 10 px bigger on every side than the content box). 
+
+There is also regular `width` and `height` properties. By default, these define the size of the content box, but you can override this behavior so that they refer to the size of the content plus the border. This is common practice now because it often makes development simpler.
+
+For now, we'll only deal with lengths in units of pixels or `px`. You can also define lengths with [different units](https://css-tricks.com/the-lengths-of-css/) including units that are relative to the font size, relative to the size of the browser size, and relative to the parent element.
+
+> Side Note: pixels don't actually refer to the pixels on your screen, they're a set length that is meant to be consistent across devices and browsers.
+
+##
+
+Defining layouts can be one of the hardest parts of mastering CSS and figuring out how to position an element just right can frustrate even seasoned developers. This is challenging because a good layout has to be adaptable to many different browser sizes that a user may have.
+
+The two prominent CSS systems for creating flexible layouts are called 'flexbox' and 'grid'. Flexbox is great for items that should be grouped together but don't need to have a particular size. Grid is great when there are very clear boundaries for where elements should be. These layouts can be used together as well for more complicated layout patterns. To learn more about these layouts and how to implement themselves you can visit [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for flexbox and [here](https://css-tricks.com/snippets/css/complete-guide-grid/) for grid.
+
```

Comment:
Should we also introduce the `display` and `position` properties? These are fundamental to understanding CSS layout